.travis.yml

```diff
@@ -34,7 +34,7 @@ script:
   - npm run sonar
 
 after_script:
-  - npm run test:coveralls
+#  - npm run test:coveralls
```

Comment:
I guess you have decided to disable this for some reason?

Reply:
Thanks, it was committed by error. I revert this change.